src/client/clouds.cpp

```diff
@@ -170,7 +170,7 @@ void Clouds::render()
 
 	// Read noise
 
-	std::vector<char> grid(m_cloud_radius_i * 2 * m_cloud_radius_i * 2); // vector<bool> is broken
+	std::vector<bool> grid(m_cloud_radius_i * 2 * m_cloud_radius_i * 2);
```

Comment:
https://github.com/minetest/minetest/pull/10660#pullrequestreview-536104315

Reply:
Yeah I remember but I couldn't find a reason why `vector<bool>` shouldn't work here and it in fact does.